c2rust-analyze/src/context.rs

```diff
@@ -164,14 +164,14 @@ impl<'tcx> GlobalAnalysisCtxt<'tcx> {
         *next_ptr_id = counter;
     }
 
-    pub fn label_field(&mut self, field: &FieldDef) {
+    pub fn assn_ptr_to_field(&mut self, field: &FieldDef) {
```

Comment:
What does `assn` mean?  `assign`?  Is that already a standard naming convention?  `assn` is only 2 characters shorter than `assign`.

Reply:
certain MIR portions receive a pointer id that is unique to them or their usage so that dataflow and permissions properties can be tracked properly, see the `AssignPointerIds` trait

Reply:
expanded names in c0f91f45